app/models/acl_dao.rb

```diff
@@ -22,18 +23,24 @@ def get_role(user_id)
     end
 
     roles = Set.new
+
+    # this logic assumes that a user can only belong to 1 daac, which is the case with dashboard.
+    daac = nil
     items.each do |item|
-      roles << "daac_curator" if item['name'].end_with?"DASHBOARD_DAAC_CURATOR"
+      if item['name'].end_with?"DASHBOARD_DAAC_CURATOR"
```

Comment:
We should log if for some reason a user is provisioned with multiple DAACs.
Check with Alicia on wording for user not provisioned.